### DISCUSSION TITLE
LUFS "vs" DB output

### DISCUSSION THREAD

**Discussion by JustCauseWhyNot** [wwmm/easyeffects#2772] [Q&A]
I hear that you don't wanna have db output louder than 78 if listening for the whole day. I wonder though how that interacts with autogain/lufs level. I ask here, because this is the only way to have system level loudness equalization on linux that I know of. If someone had in ear binaural microphones to use in order to get a pretty accurate measurement of the db output of a pair of headphones. Would they want to use other volume effecting effects to be used in order to maintain a consistent output across all applications, or would they wanna use there microphone's, and a tone generator like easyeffects builtin one that plays a consistent tone at a certain frequency in order to dial in the db output?

**Comment by wwmm** [comment]
>db output louder than 78

Is this value right? It is strange it is a positive value. And independently of the value sign the number magnitude seems too big. In any case I never researched about the best volume for health when listening audio the whole day.

As far as I remember some references like the ears pain/damage threshold are measured by playing a sign wave around `1 kHz`. Probably because our ears evolved to be more sensitive to speech frequencies. But for what you are trying to figure out  this may not be the best approach. For example when listening to audio through headphones for long periods it is usually recommended to use a crossfeed because the excessive separation of the stereo sound caused by the headphones can be tiring. I would imagine that the whole frequency spectrum has to be taken into account when dealing with things like this.

But back to our autogain something to have in mind is that the library `libebur128` it uses estimates perceived loudness. The sound intensity root mean square may or may not be more important than the loudness units when thinking about what is better for health.

**Reply by JustCauseWhyNot** [reply]
> Is this value right? It is strange it is a positive value. And independently of the value sign the number magnitude seems too big. In any case I never researched about the best volume for health when listening audio the whole day.

I think the issue is I made a mistake and said db not dba. I've heard 78dba before, but don't have source for exactly where. Here's [this](https://www.audiosciencereview.com/forum/index.php?threads/safe-listening-levels-and-headphone-voltage-power-requirements.37860/) forum post with some methodology, and this [dba chart](https://ehs.yale.edu/sites/default/files/files/decibel-level-chart.pdf) that kinda indicates 78db to be near a max you'd wanna listen too for long periods of time.

**Reply by wwmm** [reply]
>Here's [this](https://www.audiosciencereview.com/forum/index.php?threads/safe-listening-levels-and-headphone-voltage-power-requirements.37860/) forum post with some methodology

I see. Wikipedia has some information about the A-weighting https://en.wikipedia.org/wiki/A-weighting. It is a technique that tries to take into account the human ear frequency response. The same thing the EBUR128 standard used in our autogain tries to do.

>and this [dba chart](https://ehs.yale.edu/sites/default/files/files/decibel-level-chart.pdf) that kinda indicates 78db to be near a max you'd wanna listen too for long periods of time.

Ok. This is a value calculated having the `threshold of hearing` as base level. I think this threshold is defined using a sine wave of `1 kHz`. I am not sure about how easy is to make the correspondence to the dB values we see in an audio app. In both cases it is a decibel scale but the reference point is quite different. What can be seen in the fact in this case you have positive decibel values.

I think that to correlate the scales it would be necessary to player a test signal at a givel volume value and measure the intensity of the audio wave close to the ear. With this value a new decibel calculation using the `threshold of hearing` as reference would give a value that could be compared to the one in your `dba chart`.

**Reply by JustCauseWhyNot** [reply]
What is the main takeaway I should have? I'm sorry, but I need to be spoonfed information.

**Reply by wwmm** [reply]
I am not sure :smile: . Honestly I think you should just set a volume that feels comfortable without worrying too much about trying to match the values recommended in that dba chart. Without proper estimations of the sound intensity getting to your ears it will be hard to use it well.

**Reply by JustCauseWhyNot** [reply]
But what if you have a pair of in ear binaural microphones that could measure the sound?

**Reply by wwmm** [reply]
>But what if you have a pair of in ear binaural microphones that could measure the sound?

Are they calibrated in a way you can get the sound wave intensity in Watts / m² so you can compare to the reference value of `I = 10⁻¹² W/m²` that was probably used to build that dba chart?

**Reply by JustCauseWhyNot** [reply]
Hmm. I do need to calibrate them. But I don't know how I could get the watts/m2.
